pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -53,13 +53,13 @@ rec {
   # Vulkan developer beta driver
   # See here for more information: https://developer.nvidia.com/vulkan-driver
   vulkan_beta = generic rec {
-    version = "515.49.25";
-    persistencedVersion = "515.48.07";
-    settingsVersion = "515.48.07";
-    sha256_64bit = "sha256-5j+YtKaPhDxd9bcPX10ViugLMCTXEYJfod+ecn3SHWc=";
-    openSha256 = "sha256-EnZXEvic9GdcNbcvpmbDkq6YPYqypBKyEXxFJJJJpKk=";
-    settingsSha256 = "sha256-XwdMsAAu5132x2ZHqjtFvcBJk6Dao7I86UksxrOkknU=";
-    persistencedSha256 = "sha256-BTfYNDJKe4tOvV71/1JJSPltJua0Mx/RvDcWT5ccRRY=";
+    version = "525.47.04";
+    persistencedVersion = "525.60.11";
+    settingsVersion = "525.60.11";
```

Comment:
Should these be updated to 525.78.01 as well?

Reply:
https://github.com/NVIDIA/nvidia-settings/compare/525.60.11...525.78.01

Seems like they haven't added required nvml features, so that should be fine.